src/backend/replication/walsender.c

```diff
@@ -3826,9 +3826,10 @@ LagTrackerRead(int head, XLogRecPtr lsn, TimestampTz now)
 void
 GetMinReplicaLsn(XLogRecPtr* write_lsn, XLogRecPtr* flush_lsn, XLogRecPtr* apply_lsn)
```

Comment:
Do we need this function at all?
I don't see any calls

Reply:
I removed GetMinReplicaLsn function